src/nunit.analyzers.tests/TestCaseUsage/TestCaseUsageAnalyzerTests.cs

```diff
@@ -239,6 +239,85 @@ class AnalyzeArgumentHasImplicitConversion
             AnalyzerAssert.Valid<TestCaseUsageAnalyzer>(testCode);
         }
 
+        [Test]
+        public void AnalyzeArgumentIsStringConvertedToTypeWithCustomConverter()
+        {
+            var testCode = TestUtility.WrapClassInNamespaceAndAddUsing(@"
+    using System.ComponentModel;
+
+    class AnalyzeArgumentIsStringConvertedToEnum
+    {
+        [TestCase(""A"")]
+        public void Test(CustomType p) { }
+    }
+
+    [TypeConverter(typeof(CustomTypeConverter))]
+    struct CustomType { }
+    class CustomTypeConverter : TypeConverter { }");
```

Comment:
I saw the other tests had this indentation. Is this what you would still do even though there's a using statement? Is indenting something I should consider doing elsewhere?

Reply:
I think it is fine as it is.